aurelia/vscode-extension
# Fix smart-autocomplete path processing

## What was done
Issues arose stating, that the extension did not work for them.
Investigation revealed, that the path processing was the culprit.
This PR fixes that.

## Testing
- Run `au new` on a mac/linux/windows machine.
- Create a custom-element (beside app.ts/html)
- In the view type `<custom-element`
- This should reveal

#### Ubuntu 18.04
![image](https://user-images.githubusercontent.com/30693990/74590182-4c68fd00-500c-11ea-8d5a-ea07620d3771.png)

#### Windows 10
![vscode-auto-windows](https://user-images.githubusercontent.com/30693990/74590507-50e2e500-500f-11ea-9c91-00a69c614382.png)


## Todo
- [x] Verify on a windows machine as well
- [x] #110 Custom Element and View Model variables in .html 
- [x] #104 Go to Definition 
- [x] #109 in view autocomplete of viewModel vars
